app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ] It would be better if we create the `new` method of our CommentsController to handle the `GET` requests